concrete/js/build/core/user/selector.js

```diff
@@ -62,6 +63,9 @@
 
         loadUser: function(uID) {
             var my = this;
+            if(window.$clickedSelector){
+                my.$element = window.$clickedSelector;
```

Comment:
I don't think it's correct to set `my.$element` to the `$element` of another instance...

Reply:
See https://github.com/haeflimi/concrete5/pull/1

Reply:
Yeah. That looks like a more propper fix.